ci/workflows/pull-request.yml

```diff
@@ -149,25 +149,6 @@ steps:
     cancel_on_build_failing: true
     retry: *auto-retry
 
-  - label: "connector node integration test Java {{matrix.java_version}}"
```

Comment:
It is totally removed?

Reply:
Can we modify it to only run the CI when directory `java` is modified?

Reply:
This is more difficult on buildkite to do it, let us only keep the main pipeline first, and then we can study how to run this job only when the java directory is modified.

Reply:
I think we can only add `apt update` in this PR and see whether it can pass the CI or not. If it can pass, we don't have to remove the java test here. If it fails, then we remove the java test.

Reply:
You removed the job then how can we check whether `apt update` would work or not?

Reply:
The job will run when we try to merge into main.

Reply:
It looks that the job passed only add `apt update`.